### ISSUE TITLE
What (if any) extra functionality does everybody think would benefit this library?

### ISSUE THREAD

**Issue by dhaspden** [dhaspden/nestjs-stripe#69]
### Request For Feedback

Hi anybody who is reading this! Now that a few people have started to use this library I wanted to take the chance to reach out to you all. This library fulfills what it set out to do but I'd like to extend the question anyway — is there something that this library does not currently do that would be nice to have?

I toyed with the idea of webhook listeners but not sure it's worth the time investment if nobody would actually use it.

Alternatively just thumbs up this message if you think that this library is good as-is and only needs maintenance. I'll leave this up for a little bit to see if I get any responses 😃.

**Comment by vanhumbeecka** [comment]
Dunno about extending it. I use it as a simple wrapper for injecting Stripe, which was your intention from the beginning i assume. As long as the as-is version keeps being maintained, that's certainly good enough for me.

**Comment by dhaspden** [comment]
Thanks for the feedback! I plan on maintaining the current functionality indefinitely 😄

**Comment by iamjoeker** [comment]
IMO, WebHook listeners would be a very welcomed addition!

**Comment by checor** [comment]
I also agree on webhook listeners. Seems that there isn't a very clean way to do it at the moment, so using this library for webhooks would be nice!

**Comment by gabimor** [comment]
I stated using this package but switched to `@golevelup/nestjs-stripe` just because of the webhook support.
BUT its an actual nightmare to get it working there, after a few hours and still no success..... :(
So if you can come up with a simpler or even a better documented solution, that would be great!
